tests/ga4gh/drs/test_server.py

```diff
@@ -1,4 +1,5 @@
 from flask import json
+from copy import deepcopy
```

Comment:
Please structure the imports according to Python's [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) styleguide, and each section in alphabetical sort order as suggested [here](https://stackoverflow.com/a/20763446). 

So basically the generals structure is to divide imports into three blocks, each separated by an empty line:
1. Standard library imports (e.g., `copy`, `os`, `sys` etc., anything you can import without installing extra packages)
2. Third party imports (anything from packages that need to be installed except for the local application)
3. Local application (i.e., all imports from DRS-Filer)

So in this example it should be:

```py
from copy import deepcopy

from flask import json
from foca.models.config import MongoConfig

from drs_filer.errors.exceptions import (
    ...
)
```

So while you are at it, please use the chance to add an empty line between the `foca` and `drs_filer` imports as well, even though you haven't changed anything here. :)